src/webview/webview.rs

```diff
@@ -135,14 +135,45 @@ impl Window {
                         {
                             log::error!("Verso failed to send AllowNavigationRequest to controller: {error}")
                         } else {
-                            // We will handle a ControllerMessage::OnNavigationStartingResponse
+                            // We will handle a ToVersoMessage::OnNavigationStartingResponse
                             // and send ConstellationMsg::AllowNavigationResponse there if the call succeed
                             return;
                         }
                     }
                 }
                 send_to_constellation(sender, ConstellationMsg::AllowNavigationResponse(id, true));
             }
+            EmbedderMsg::WebResourceRequested(_webview_id, request, sender) => {
```

Comment:
I am wondering if we should add a feature flag in the future as this kind of logic growing up will increasing maintenance effort.

Reply:
I think it might be hard to keep everything related to the webview controllers in a feature flag as it's quite integrated, but we could try that in a future PR